src/bme680.toit

```diff
@@ -278,36 +279,37 @@ class Driver:
 
     temp_adc >>= 4
 
+    // Datasheet, page 17.
     var1 := ((temp_adc / 16384.0) - (par_T1_ / 1024.0)) * par_T2_
-    var2 := (((temp_adc / 131072.0) - (par_T1_ / 8192.0)) *
-      (temp_adc / 131072.0) - (par_T1_ / 8192.0)) * 
-      (par_T3_ * 16.0)
+    var2 := (((temp_adc / 131072.0) - (par_T1_ / 8192.0))
+      * (temp_adc / 131072.0) - (par_T1_ / 8192.0))
+      * (par_T3_ * 16.0)
     t_fine_ = var1 + var2
     temp_comp_ = t_fine_ / 5120.0
-    
-
 
   wait_for_measurement_:
-    100.repeat: // If maximum gas heater wait time is used (4096ms) it takes roughly 90 iterations before the measurement is done.
+    // If maximum gas heater wait time is used (4096ms) it takes roughly
+    // 90 iterations before the measurement is done.
+    100.repeat:
       val := reg_.read_u8 MEAS_STATUS_REG_
-      if (val & 0b1000_0000) >> 7 == 1: //Last bit indicates new mesurement values ready
-        return
+      // MSB bit indicates new measurement values ready.
+      if (val >> 7) & 1 == 1: return
```

Comment:
Why is this better? Is it faster? Or just more compact code?

Reply:
It's less error-prone.
The `0b1000_0000` needs to aligned with the `>> 7`.

It's not faster or better. Just easier to read and maintain.

Reply:
Then it is better. :-)